.github/workflows/continuous-integration.yaml

```diff
@@ -0,0 +1,156 @@
+name: Continuous integration
+
+on:
+  push:
+    branches: [ master, release* ]
+  pull_request:
+    branches: [ master, release* ]
```

Comment:
```suggestion
    branches: [ master, release*, feat* ]
```

Reply:
If there is a feature branch (for some reason) it must be merged via a PR.  No need to run _another_ job against the branch itself.